lib/cell/templates.rb

```diff
@@ -30,7 +30,7 @@ def create(prefix, view, options)
     # {["comment/row/views", comment/views"]["show.haml"] => "Tpl:comment/view/show.haml"}
     class Cache
       def initialize
-        @store = {}
+        @store = Hash.new({})
```

Comment:
Oh fuck, is that true? I didn't know that! That's pretty hectic. @timoschilling - why do you introduce such bugs in our gem? :laughing: